core/services/gas/chain_specific.go

```diff
@@ -2,12 +2,11 @@ package gas
 
 import "math/big"
 
-// chainSpecificIsUsableTx allows for additional logic specific to a
-// particular chain that determines whether a transction should be used for gas
-// estimation
-func chainSpecificIsUsableTx(tx Transaction, minGasPriceWei, chainID *big.Int) bool {
-	if isXDai(chainID) {
-		// GasPrice 0 on most chains is great since it indicates cheap/free transctions.
+// chainSpecificIsUsableTx allows for additional logic specific to a particular
+// chain that determines whether a transaction should be used for gas estimation
+func chainSpecificIsUsableTx(tx Transaction, minGasPriceWei *big.Int, chainType string) bool {
```

Comment:
Could also be moved to a method or function on the `chain` like `chain.IsUsableTx`.

I don't know if this is cleaner or not.

Reply:
There might be a clean way to do this, but the naive approach would invert some import relationships, so I simplified it by passing the `Config` all the way down instead.

Reply:
These actually work nicely as methods on `*Transaction`, and can then drop the `Tx` suffix as well. Updated.